Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -22,34 +22,35 @@ struct SettingsView: View {
     
     var body: some View {
         NavigationStack {
+            // swiftlint:disable:all line_length
```

Comment:
Please switch all of these to `// swiftlint:disable:next line_length`
Which only disables the next line for SwiftLint (as opposed to disabling the entire file)